### DISCUSSION TITLE
[Feature Request] Add option to unmute with PulseVolume's `increase_vol`/`decrease_vol`

### DISCUSSION THREAD

**Discussion by SpyrosRoum** [qtile/qtile#3970] [Ideas] [answered]
I think the title pretty self-explanatory but essentially I'd like to have the option to instruct pulse_volume to also unmute itself with `increase_vol`/`decrease_vol`.

Similar to the `--unmute` option of pulsemixer (e.g. `pulsemixer --change-volume -5 --unmute`).

**Comment by m-col** [comment, marked as answer]
Would this do?: https://github.com/qtile/qtile/pull/3972

**Reply by SpyrosRoum** [reply]
I didn't  know you can chain commands together like that!
Indeed that would do it

**Comment by AbdelrhmanUZaki** [comment]
There is any easy way


```
# Volume control bindings (using function keys and XF86Audio keys)
    Key([], "XF86AudioRaiseVolume", lazy.spawn("pactl set-sink-volume @DEFAULT_SINK@ +5%"),desc="Raise volume"),
    Key([], "XF86AudioLowerVolume", lazy.spawn("pactl set-sink-volume @DEFAULT_SINK@ -5%"),desc="Lower volume"),
    Key([], "XF86AudioMute", lazy.spawn("pactl set-sink-mute @DEFAULT_SINK@ toggle"),desc="Mute and Unmute volume toggle"),
```
